templates/catalog/book/bookFilesContentAndDownload.tpl

```diff
@@ -0,0 +1,34 @@
+{**
+ * templates/catalog/book/bookFiles.tpl
+ *
+ * Copyright (c) 2014 Simon Fraser University Library
```

Comment:
Copyrights on new code should be extended into 2015. (Use 2014-2015 here, and 2003-2015 in the next line.) Frankly I'm not particularly knowledgeable about the legal implications of the dates, so I'm just trying to keep them generally consistent.